pkg/kinflate/app/application.go

```diff
@@ -30,6 +30,8 @@ import (
 type Application interface {
 	// Resources computes and returns the resources for the app.
 	Resources() (resource.ResourceCollection, error)
+	// SemiResources computes and returns the resources without name hash and name reference for the app
+	SemiResources() (resource.ResourceCollection, error)
```

Comment:
This doesn't have to be public.

Reply:
`SemiResources` is called by the sub applications. They are not available if set to private.